rbac/ledger_sync/deltas/updating.py

```diff
@@ -158,30 +129,38 @@ def _update_state(database, block_num, address, resource):
 
 
 def _update_legacy(database, block_num, address, resource):
-    data_type = addresser.address_is(address)
     try:
-        data = {
-            "id": address,
-            "start_block_num": int(block_num),
-            "end_block_num": int(sys.maxsize),
-            **resource,
-        }
-
-        table_query = database.get_table(TABLE_NAMES[data_type])
-        query = table_query.get(address).replace(
-            lambda doc: r.branch(
-                # pylint: disable=singleton-comparison
-                (doc == None),  # noqa
-                r.expr(data),
-                doc.merge(resource),
+        data_type = addresser.address_is(address)
```

Comment:
This is probably too much to address here, but how much do we have to support legacy on this? I assume there's a reason we can't throw it away yet.

Reply:
for posterity's sake, can we say how we resolved this? I believe in this case we removed the offending code?

Reply:
We did a global search+replace.

Reply:
Removes is_address in favor of get_address_type.